### ISSUE TITLE
classe player

### ISSUE THREAD

**Issue by dunindunin** [trouba/grand-carpet#10]
si on pouvait créer une classe player qu'on instancierait avec les boutons de controle (keycode)

et on aurait juste une methode joueur.update() à insérer dans Game
comme celoa, on pourait facilement faire une version deux joueurs ou le support du gamepad...

**Comment by riddletom** [comment]
Les controles doivent etre differents, mais peuvent eventuellement etre un
parametre d entree de la classe

**Comment by dunindunin** [comment]
oui, je proposais de mettre en argument dans le constructeur les keycodes:
puis tu fais:
var upKey = game.input.keyboard.addKey(argument1);
etc.

et après (dans le update):
if (upKey.isDown)

**Comment by dunindunin** [comment]
ou en argument (magie de javascript), tu mets carrément la fonction de test de chaque direction...
